src/main/resources/init_db.sql

```diff
@@ -0,0 +1,9 @@
+CREATE DATABASE `taxi_db`;
+
+CREATE TABLE `manufacturers` (
+                                 `id` bigint NOT NULL AUTO_INCREMENT,
+                                 `name` varchar(255) DEFAULT NULL,
+                                 `country` varchar(255) DEFAULT NULL,
+                                 `is_deleted` tinyint NOT NULL DEFAULT '0',
+                                 PRIMARY KEY (`id`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb3
```

Comment:
Please use an [Intellij code formatting](https://www.jetbrains.com/help/idea/reformat-and-rearrange-code.html) to format this file properly, this will increase readability of your solution.